test/transcoding/FPGALoopAttr.ll

```diff
@@ -0,0 +1,157 @@
+; RUN: llvm-as < %s > %t.bc
+; RUN: llvm-spirv %t.bc -o - -spirv-text | FileCheck %s --check-prefix=CHECK-SPIRV
+
```

Comment:
I think if you add translation back to LLVM IR and check the IR, this test will cover other test cases in this PR, right?
If so, you could remove other tests.

Reply:
Done